src/main.js

```diff
@@ -63,5 +63,33 @@ toDoInput.addEventListener('keypress', (e) => {
 const label = document.querySelectorAll('.task');
 label.forEach((input, index) => input.addEventListener('change', () => {
   tasksArray[index].description = input.value;
-  storage(tasksArray);
-}));
\ No newline at end of file
+  Storage(tasksArray);
+}));
+// const list  = document.querySelector('#task');
+const complete = () => {
+  const check = document.querySelectorAll('input[type=checkbox]');
+  check.forEach((input, index) => input.addEventListener('change', () => {
+    if (input.checked) {
+      tasksArray[index].completed = true;
+      // list[index].style.textDecoration = 'line-through';
+      // list[index].style.color = 'grey';
+    } else {
+      tasksArray[index].completed = false;
+      //    list[index].style.textDecoration = 'none';
+      //    list[index].style.color = 'black';
+    }
+    Storage(tasksArray);
+  }));
+};
+
+const clear = document.querySelector('#clear-button');
+clear.addEventListener('click', () => {
+  const completed = tasksArray.filter((data) => data.completed === true);
```

Comment:
- [ ] The `clear completed` functionality isn't working properly, kindly fix this issue 👍 

![Animation](https://user-images.githubusercontent.com/50721479/193337331-3f863d5a-b136-42d8-ac9c-accfa3db0849.gif)

Reply:
hello manel 
can you please help me out with how to fix it?